pmd/pmd.xml

```diff
@@ -8,46 +8,46 @@
   </description>
   <exclude-pattern>.*/org.terasology.protobuf/.*</exclude-pattern>
 <!-- inline <rule ref="rulesets/java/basic.xml"/>  -->
-  <rule ref="category/java/errorprone.xml/AvoidBranchingStatementAsLastInLoop" deprecated="true" />
```

Comment:
why can we just remove the deprecation note for this and the others?
was the deprecation related to not having them inlined? :thinking:

Reply:
yes, the way it was organized is deprecated. i left the comments  so one can see in the beginning where the rules came from.

Reply:
@jdrueckert anything missing or it can be merged as is?